mungegithub/mungers/approval-handler.go

```diff
@@ -202,13 +202,27 @@ func (h ApprovalHandler) unwrapAliases(ownerSet sets.String) sets.String {
 	return aliases.Expand(ownerSet)
 }
 
+func getRandomizedKeys(approverOwnersfiles map[string]sets.String) []string {
+	sliceOfKeys := make([]string, 0, len(approverOwnersfiles))
+	for approver := range approverOwnersfiles {
+		sliceOfKeys = append(sliceOfKeys, approver)
+	}
+	sort.Strings(sliceOfKeys)
+	order := rand.Perm(len(approverOwnersfiles))
```

Comment:
Wait, the seed is the length of approverOwnersFiles? So if this set of file is the same, it will have the same seed?